ms-campus-kid-grades/src/db_postgres/app.py

```diff
@@ -0,0 +1,101 @@
+import json
+from flask import Flask
+from flask import request
+import flask_sqlalchemy
+import os
+
+#user = os.environ['postgres']
+#password = os.environ['test123']
+#host = os.environ['postgres']
+#port = os.environ['5432']
+#database = os.environ['estudiantes_sqlalchemy']
+
+#DATABASE_CONNECTION_URI = f'postgresql+psycopg2://{user}:{password}@{host}:{port}/{database}'
+DATABASE_CONNECTION_URI = 'postgresql+psycopg2://postgres:test123@localhost:5432/estudiantes_sqlalchemy'
+
+db = flask_sqlalchemy.SQLAlchemy()
+
+class Estudiante(db.Model):
+    __tablename__ = 'estudiante'
+    id = db.Column(db.Integer, primary_key = True)
+    correo_estudiante = db.Column(db.String(100), nullable=False, unique=True)
+    nombre_estudiante = db.Column(db.String(100), nullable=False,)
+    id_facultad = db.Column(db.Integer, nullable=False, unique=True)
+    # created_at = db.Column(db.Datetime(), default=datetime.now())
```

Comment:
Why do we need this comment?

Reply:
Lately we can use it in another db to obtain times on insertions.